src/routes/Account/modules/account.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
```

Comment:
does flow detect `Promise` correctly if we're using `q` instead of a Promise polyfill?